cpp/cmake/modules/cuda_configuration.cmake

```diff
@@ -138,6 +138,9 @@ function(setup_cuda_architectures)
         message(FATAL_ERROR "Unrecognized CUDA architecture: ${CUDA_ARCH}")
       endif()
     endforeach()
+    if("103" IN_LIST CMAKE_CUDA_ARCHITECTURES_CLEAN)
```

Comment:
Why is this needed, Is this a WAR? Can we just use 103? Thx

Or please help to add some comments.